transcription/transcription.go

```diff
@@ -38,6 +45,49 @@ func msgHeaders(from string, to []string, subject string) string {
 	return strings.Join(msgHeaders, "\r\n")
 }
 
+// StartTranscription transcribes a given file using Sphinx.
+// File name should be in "name.wav" format.
+func StartTranscription(fileName string, command string) (string, error) {
+
+	cmd := exec.Command("java", "-jar", command, fileName)
+	_, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", err
+	}
+	outputFileName := "/Sphinx/files/" + fileName + "-json.txt"
```

Comment:
`.json` is a ascii text format. I would use that but it's not a big deal either way.